.circleci/config.yml

```diff
@@ -55,6 +68,7 @@ jobs:
 workflows:
   commit:
     jobs:
+      - controller-test
```

Comment:
> A quicker way to do this would be on the docker executor, rather than "machine". You don't need sudo or full permissions, and the boxes boot about 1 minute faster:

I wanted to go this route, but couldn't because of [this issue](https://github.com/RespiraWorks/VentilatorSoftware/pull/181#discussion_r415466618). Let me know if you think of any way around this.